### PR TITLE
refactored footer and styled the links

### PR DESCRIPTION
**Pull request by VladyslavPetskovych**
refactored footer and styled the links

[Home page] The buttons are not centered at the home page footer. #616

image

### CODE REVIEW

File: src/containers/layout/footer/Footer.styles.ts [hunk 1]
```diff
@@ -1,19 +1,24 @@
 export const styles = {
   footer: {
     backgroundColor: 'primary.900',
-    color: 'primary.50',
-    marginTop: 'auto'
```
Comment (tsivkadi) [inline src/containers/layout/footer/Footer.styles.ts:5]: why did you removed this style

Comment (tsivkadi) [reply src/containers/layout/footer/Footer.styles.ts:5]: and you don't need to create another pull request to make some changes, just push one more commit in the same pr

Comment (VladyslavPetskovych) [reply src/containers/layout/footer/Footer.styles.ts]: Thanks

Comment (VladyslavPetskovych) [reply src/containers/layout/footer/Footer.styles.ts]: > why did you removed this style

my bad(

File: src/containers/layout/footer/Footer.styles.ts [hunk 2]
```diff
@@ -6,14 +6,20 @@ export const styles = {
...
     display: 'flex',
+    position: 'relative',
     flexDirection: { xs: 'column-reverse', sm: 'row' },
     justifyContent: { xs: 'space-between', sm: 'space-around' },
     alignItems: 'center',
     py: { xs: '12px', sm: '26px' }
   },
+  title: {
+    position: { xs: 'static', sm: 'absolute' },
+    marginLeft: { xs: '0', sm: '-60vw' }
```
Comment (MickhaelSh) [inline src/containers/layout/footer/Footer.styles.ts:17]: why margin -60vw? do container max-width as in Figma , position title absolute and left 0

File: src/containers/layout/footer/Footer.styles.ts [hunk 3]
```diff
@@ -6,14 +6,20 @@ export const styles = {
...
     display: 'flex',
+    position: 'relative',
     flexDirection: { xs: 'column-reverse', sm: 'row' },
     justifyContent: { xs: 'space-between', sm: 'space-around' },
     alignItems: 'center',
     py: { xs: '12px', sm: '26px' }
   },
+  title: {
+    position: { xs: 'static', sm: 'absolute' },
+    left: { xs: '0', sm: '0' }
```
Comment (MickhaelSh) [inline src/containers/layout/footer/Footer.styles.ts:17]: no need for xs sm etc if it`s always 0

File: src/containers/layout/footer/Footer.styles.ts [hunk 4]
```diff
@@ -6,14 +6,20 @@ export const styles = {
...
+  },
   links: {
     display: { xs: 'flex', sm: 'block' },
     flexDirection: { xs: 'row', sm: 'column' },
+    marginLeft: { xs: '0', sm: '48px' },
```
Comment (MickhaelSh) [inline src/containers/layout/footer/Footer.styles.ts:22]: why do we need a margin if we need them to be in center

File: src/containers/layout/footer/guest-footer/GuestFooter.tsx [hunk 5]
```diff
@@ -15,7 +15,7 @@ const GuestFooter = () => {
 
   return (
     <Container sx={styles.container}>
-      <Typography variant={TypographyVariantEnum.Caption}>
+      <Typography sx={styles.title} variant={TypographyVariantEnum.Caption}>
```
Comment (FryDay969) [inline src/containers/layout/footer/guest-footer/GuestFooter.tsx:18]: You may also move "variants" into the styles file.

File: src/containers/layout/footer/Footer.styles.ts (hunk 4 above)
Comment (VladyslavPetskovych) [reply src/containers/layout/footer/Footer.styles.ts:22]: becouse of overlaping

image

**Pull request closed without merging**